templates/user/linux/default/installer.erb

```diff
@@ -0,0 +1,11 @@
+user_check() {
+  getent passwd "$1" > /dev/null 2>&1
+}
+
+user_create() {
+  useradd "$1"
```

Comment:
hmm, yeah I guess so, I'll use `--system` to make it more explicit (the `-r` to mean 'system user' is ... confusing)